akka-actor/src/main/scala-2.13+/akka/util/ByteString.scala

```diff
@@ -754,10 +788,21 @@ sealed abstract class ByteString
    */
   protected[ByteString] def toArray: Array[Byte] = toArray[Byte]
 
-  override def toArray[B >: Byte](implicit arg0: ClassTag[B]): Array[B] = iterator.toArray
+  final override def toArray[B >: Byte](implicit arg0: ClassTag[B]): Array[B] = {
```

Comment:
mima complains that the method became final, but since the class is sealed that seems fine...
```
[error]  * method copyToArray(java.lang.Object,Int)Int in class akka.util.ByteString is declared final in current version
[error]    filter with: ProblemFilters.exclude[FinalMethodProblem]("akka.util.ByteString.copyToArray")
[error]  * method toArray(scala.reflect.ClassTag)java.lang.Object in class akka.util.ByteString is declared final in current version
[error]    filter with: ProblemFilters.exclude[FinalMethodProblem]("akka.util.ByteString.toArray")
```